### DISCUSSION TITLE
Label ggplotly tidak muncul sesuai kode

### DISCUSSION THREAD

**Discussion by AGelfP** [teamalgoritma/community#550] [Q&A] [answered]
Saya berencana memberikan label untuk setiap titik yang ada pada line plot saya. Terlampir berikut screenshot pengerjaan saya:
![image](https://user-images.githubusercontent.com/122977451/219948799-e06c3667-69f7-4266-ae7e-1c8a45e3f2b3.png)

Bisa dilihat bahwa label menunjukkan text yang seharusnya tidak ada, yaitu "year (yang berawal dengan huruf kecil)", "Average_Views", dan "category_id". Kira-kira cara menghilangkannya harus diapakan ya? 

Terima kasih sebelumnya

**Comment by divakartika** [comment, marked as answer]
Dear @AGelfP ,

Jika hasil `glue()` dimasukkan ke dalam kolom baru dalam data `for_plot2` apakah terjadi hal yang sama?

**Reply by AGelfP** [reply]
Halo Kak, iya kemarin sudah dikasih solusi yang serupa, sudah terselesaikan Kak masalahnya. Terima kasih ya

**Reply by divakartika** [reply]
Siap baik sama-sama, Pak :)

Best Regards,
Team Algoritma
